packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -361,13 +366,15 @@ export function transformPrivateNamesUsage(
   ref,
   path,
   privateNamesMap,
-  loose,
+  privateFieldsAsProperties,
```

Comment:
Consider change the signature to `{ privateFieldsAsProperties: boolean }` so we can pass `noDocumentAll` later. This function is not a public interface so it should be okay.